### ISSUE TITLE
Scrap trending videos from the past

### ISSUE THREAD

**Issue by timle3** [DataSnaek/Trending-YouTube-Scraper#3]
Hi,

I came across this youtube scraper from kaggle and think it's the perfect script to get relevant information about current trending videos on youtube. However, I was wondering if there's a way to get trending videos from the past? The kaggle dataset is not up to date and I would like to create a new dataset for a project that I'm doing for school. Is there any way to do that with this script?

**Comment by devianpctek** [comment]
> Hi,
> 
> I came across this youtube scraper from kaggle and think it's the perfect script to get relevant information about current trending videos on youtube. However, I was wondering if there's a way to get trending videos from the past? The kaggle dataset is not up to date and I would like to create a new dataset for a project that I'm doing for school. Is there any way to do that with this script?

I don't think you can do that, but try searching here [Youtube API](https://developers.google.com/youtube/v3/docs/videos/list), at least I didn't find any "date" related that you can use to get the videos

**Comment by rsrishav** [comment]
Not possible.
